### PR TITLE
build nightly

### PR DESCRIPTION
**Pull request by alexandercbooth**
Now that we have some basic tests, I thought it might be good to get a nightly build going.

I haven't configured this before so we'll see 🙃. 

The intent is for builds to still happen on every commit like normal, but then also automatically happen once a night at 12:00am UTC

Also haven't used cron for about a 100 years 😬

### CODE REVIEW

**Comment by rgbkrk** [comment]
Wahoo!

File: .circleci/config.yml [hunk 1]
```diff
@@ -39,7 +39,22 @@ jobs:
...
+
+workflows:
+  version: 2
+  commit:
+    jobs:
+      - build
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
```
Comment (rgbkrk) [inline .circleci/config.yml]: It's usually friendly to other systems if you do repeated runs that are not "on the hour". 😄

Comment (alexandercbooth) [reply .circleci/config.yml]: Wow I hadn't even considered that, good thinking! Will change 👍

**Comment by rgbkrk** [comment]
Thanks for setting this up! I had to run away right before clicking merge and you updated the cron job in the meantime. Awesome. 😄

**Comment by alexandercbooth** [comment]
My pleasure 😄

**Pull request merged**